sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -2985,10 +2985,6 @@ _ZN4sycl3_V121__isgreaterequal_implEdd
 _ZN4sycl3_V121__isgreaterequal_implEff
 _ZN4sycl3_V122accelerator_selector_vERKNS0_6deviceE
 _ZN4sycl3_V128verifyUSMAllocatorPropertiesERKNS0_13property_listE
-_ZN4sycl3_V13ext5intel12experimental15online_compilerILNS3_15source_languageE0EE12compile_implENS0_6detail11string_viewERKSt6vectorIS8_SaIS8_EESt4pairIiiENS0_4info11device_typeENS3_11device_archEbS8_RPvSK_
```

Comment:
Should we wait a couple more months and remove this during ABI-breaking window? Otherwise, it will break the ABI during minor release and QA will come haunt us :|

Reply:
Ahh.. I just re-read the PR description: "The decision has been made to remove it, without waiting for an ABI breaking window." and it answers my question.

Reply:
The ABI compatibility requirement doesn't not apply **any experimental API**!

Reply:
I had the same question but was informed that it can be removed immediately.